widgets/textinput/editor_test.go

```diff
@@ -330,6 +330,7 @@ func TestFieldEditor(t *testing.T) {
 		wantContent string
 		wantCurIdx  int
 		wantErr     bool
+		mutations   int
```

Comment:
The name mutations does not make me think of the onChange handler. Can we choose a more closely related name. E.g. we could call it wantOnChangeCalls instead.